rbac/management/group/view.py

```diff
@@ -147,13 +147,18 @@ def get_serializer_class(self):
         return GroupSerializer
 
     def protect_default_groups(self, action):
```

Comment:
Will this protect against principals being added to the group as well?

Reply:
No it doesn't. Should it?

Reply:
Yeah, that's called out in the acceptance criteria:

> As any API user, I should not be able to modify any "admin_default" groups, and should receive a validation error (not updating the group or adding/removing principals).

Of note: we _should_ have this in place for the `platform_default` group as well with regard to disallowing principal association, since these two groups have implicit access only, but we don't currently do that. Might be worth logging something to also have this check on `platform_default`.

Reply:
Sounds perfect!